### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by JHasty369** [samyk/opensesame#12]
I belive you made a mistake because you said inexpensive.  Last time I checked these were in excess of 300 $.

**Comment by samyk** [comment]
As of the time of writing, they were inexpensive. Ironically they increased in price after releasing this project. Thanks for the note, will update.

**Comment by JHasty369** [comment]
yeah wonder why  one of the biggest questions in all of humanity.
